clang/test/ParserHLSL/hlsl_raw_buffer_attr.hlsl

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.0-compute -x hlsl -ast-dump -o - %s | FileCheck %s

```

Comment:
Presumably we have tests somewhere for resource handles that don't have raw_buffer on them, and they just don't show up here because they're not impacted at all by this new attribute?

Reply:
Yep, that would be `clang/test/ParserHLSL/hlsl_resource_handle_attrs.hlsl`.